#include "PaxosKVServiceImpl.hpp"
int PaxosKVServiceImpl::Get(const ::paxoskv::GetReq & oReq, ::paxoskv::GetResp & oResp) {
    return -1;
}
int PaxosKVServiceImpl::Set(const ::paxoskv::SetReq & oReq, ::paxoskv::SetResp & oResp) {
    return -1;
}
int PaxosKVServiceImpl::Del(const ::paxoskv::DelReq & oReq, ::paxoskv::DelResp & oResp) {
    return -1;
}
